style.css

```diff
@@ -0,0 +1,187 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+
+html {
+  box-sizing: border-box;
+}
+
+* *::before *::after {
+  box-sizing: inherit;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  margin: 10px;
+  padding: 10px;
+  border: 20px;
+  background-image: url(images/Header.svg);
+  background-repeat: none;
+  max-height: 100vh;
+  max-width: auto;
+}
+
+a {
+  color: #344563;
+  font-family: 'Poppins', sans-serif;
```

Comment:
- [ ] You did a great job implementing most of the changes but the styling of your portfolio looks different from the Figma design. Kindly delete the borders with the different colors that you added.
- [ ] Please fix the background position and the margin.
- [ ] The Hamburger menu is still not displaying

![review](https://user-images.githubusercontent.com/53347966/219787394-04641263-b397-47b6-addc-7b6ac510497a.JPG)

Reply:
Hello @algerina, thank you for your comments. I'm so sorry, I forgot to erase the borders.  I have a question, on my desktop I can see the hamburger menu. Do you have any suggestion?
<img width="435" alt="image" src="https://user-images.githubusercontent.com/122847613/219791384-b38c79f5-a367-4d29-8e3c-c17b84edf434.png">

Reply:
@oovillagran  the hamburger menu is not displaying because you have put a display: none on the container that contains the hamburger menu.